backend/src/rtp.rs

```diff
@@ -416,6 +416,21 @@ pub struct Packet<T> {
     serialized: T,
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum PacketError {
+    #[error("Invalid payload range in header: {0}")]
+    InvalidPayloadRange(usize),
+
+    #[error("Invalid TCC sequence number range: {0}")]
+    InvalidTccSeqNumRange(usize),
+
+    #[error("Invalid marker value")]
+    InvalidMarker,
```

Comment:
I don't see any of these used. Would it be better to have errors like PacketError::DecryptFailure|EncryptFailure instead? And then the PacketError::PacketError wouldn't be needed.

Reply:
So should this error type only consist two error scenarios ?